src/Neo.Compiler.MSIL/MSIL/Converter.cs

```
@@ -1152,12 +1135,8 @@ private int ConvertCode(ILMethod method, OpCode src, NeoMethod to)
                     break;
 
                 default:
-#if WITHPDB
                     logger.Log("unsupported instruction " + src.code + "\r\n   in: " + to.name + "\r\n");
-                    break;
-#else
                     throw new Exception("unsupported instruction " + src.code + "\r\n   in: " + to.name + "\r\n");
```

This line should be removed too?

---

Maybe the log, but not the exception, isn't it? @lightszero

---

@lightszero we should throw the exception or only log it?

---

For me, it have sense to stop the conversion if one instruction is unsupported.

---

I think should throw for here,skip this could be got more error..